docs/explanation/archive.rst

```diff
@@ -0,0 +1,348 @@
+Ubuntu Package Archive
+======================
+
+:term:`Linux` :term:`Distributions <Distribution>` like :term:`Ubuntu` use :ref:`Repositories`
+to hold :term:`Packages <Package>` you can install on target machines. :term:`Ubuntu`
+has several :ref:`Repositories` that anyone can access. The *Ubuntu Package Archive* 
+hosts :term:`Debian` :term:`Binary Packages <Binary Package>` (``.deb`` files) and
+:term:`Source Packages <Source Package>` (``.dsc`` files). On :term:`Ubuntu` installations,
+the *Ubuntu Package Archive* is configured as the default source for the :term:`APT`
```

Comment:
I think here is a greater discussion that I would like for you and @keirthana to decide. 
When does it make sense to reference a term? I initially wanted a Wikipedia style page where a term is referenced, every time it occurs, but I agree that this can be hard to read.

Reply:
Wikipedia only tends to reference the first appearance of a term, I think :thinking: 
I'd tend to overlook a second reference if it occurred further down the page (especially if the page is long or if it makes sense that readers might want to remind themselves of the meaning at that point) since Wikipedia sometimes also does this, but it should be the exception rather than the rule.

Reply:
Resolved with 6cbf3120a572322bf619fd3c28f2f6d4075b21ba